From 84685930d5cfc7005f3414601230a23e28a18f59 Mon Sep 17 00:00:00 2001
From: Thomas Loimer <thomas.loimer@tuwien.ac.at>
Date: Tue, 5 Sep 2023 22:02:55 +0200
Subject: [PATCH 1/3] Enable error message for missing libraries

Move the check for missing libraries to a place where the queried
variables are set.
---
 configure.ac | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/configure.ac b/configure.ac
index 035b9d7..47452c5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -95,9 +95,6 @@ dnl AC_CHECK_PROG(XDG_MENU, [xdg-desktop-menu], [xdg-desktop-menu])
 AC_SEARCH_LIBS([pow], [m])dnl
 dnl AC_SEARCH_LIBS([deflate], [z])  # libz is not needed.
 AC_SEARCH_LIBS([FcInit], [fontconfig])
-AS_IF([test "x$ac_cv_search_pow" = xno \
-	|| test "x$ac_cv_search_png_read_info" = xno],
-    [AC_MSG_ERROR([Necessary libraries not found.])])
 
 
 # Checks for header files.
@@ -187,6 +184,10 @@ PostScript files. To disable calling ghostscript, set to the empty string,
 AC_SUBST([gsman])
 AM_SUBST_NOTMAKE([gsman])
 
+AS_IF([test "x$ac_cv_search_pow" = xno \
+	|| test "x$ac_cv_search_png_read_info" = xno],
+    [AC_MSG_ERROR([Necessary libraries not found.])])
+
 
 # Checks for typedefs, structures, and compiler characteristics.
 AC_C_BIGENDIAN
-- 
2.40.1

